src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,23 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    private static final String START_SYMBOL = "w";
+
+    public static String[] readFromFile(String fileName) {
+        try {
+            return Files.readAllLines(new File(fileName).toPath())
+                    .stream()
+                    .flatMap(line -> Arrays.stream((line + " ").toLowerCase().split("[.!?]* ")))
```

Comment:
- why do we use (line + " ") here?
- try to replace "[.!?]* " with "\\W+"

Reply:
If we don't use "\\W+" we have to add an extra space so that the dot in the end of the file is not attached to the word after parsing